### PR TITLE
process.py: Increase bitbake timeout and add logs

### PR DESCRIPTION
**Pull request by chaitu236**
Got 10 good runs with no failures.

@ni/rtos

### CODE REVIEW

**Comment by chaitu236** [comment]
@amstewart Turns out the maxuiwait needn't be changed.
Also, upstream has been consistently increasing timeout from 5s -> [30s](https://bugzilla.yoctoproject.org/show_bug.cgi?id=12116) -> [60s](https://github.com/openembedded/bitbake/commit/e567743e70f426786ae54dcb5ab550748d9266e4) because of this issue. So I think 300s will be accepted with these changes without needing to make it a configurable variable.

**Review by amstewart — APPROVED** [review]
OK. Give it a shot.

File: lib/bb/server/process.py [hunk 1]
```diff
@@ -390,12 +390,19 @@ def __init__(self, connection, recv):
         self.connection = connection
         self.recv = recv
 
+        self.reply_wait = 5
```
Comment (shruthi-ravi) [inline lib/bb/server/process.py]: Perhaps this is too short a time for polling and will be a lot of messages. How about sticking to the original 30s for this?

Comment (chaitu236) [reply lib/bb/server/process.py]: A lot of messages in cases where it's waiting for a long time is justified and is maybe a good way to attract attention? Per https://bugzilla.yoctoproject.org/show_bug.cgi?id=12116, 5s timeout itself shouldn't happen too frequently. We've been hitting 10s timeout 50% of time so I'm ok with making it 10s. Is that ok or would you still prefer 30s?

Comment (shruthi-ravi) [reply lib/bb/server/process.py]: 10s is fine.

**Comment by chaitu236** [comment]
Update 2: Changed logging time from 5s to 10s.

**Review by shruthi-ravi — APPROVED** [review]

**Comment by shruthi-ravi** [comment]
@amstewart, is the intent to pull these commits to our repo while @chaitu236 pursues upstream integration? Or should we wait to merge until Chaitanya has an update on the upstream status? How does this work?

**Comment by chaitu236** [comment]
https://lists.openembedded.org/g/bitbake-devel/message/12061

**Comment by amstewart** [comment]
> @amstewart, is the intent to pull these commits to our repo while @chaitu236 pursues upstream integration? Or should we wait to merge until Chaitanya has an update on the upstream status? How does this work?

Since we already have a workaround in the mainline, we can wait until his patch is accepted upstream. Upstream might require changes and it will be simpler to just pull the final patchset once it is approved.

**Comment by amstewart** [comment]
@chaitu236 It doesn't seem like these commits ever made their way into hardknott. But we also haven't really had issues with the bitbake server timing out in the hardknott pipelines.

So should we just drop this PR?

**Comment by chaitu236** [comment]
> @chaitu236 It doesn't seem like these commits ever made their way into hardknott. But we also haven't really had issues with the bitbake server timing out in the hardknott pipelines.
> 
> So should we just drop this PR?

Yes, I'll drop the PR

**Pull request closed without merging**